javascript-practice/src/js/app.js

```diff
@@ -16,7 +16,8 @@ export default class App {
       view.bindFormLogin(controller.handleFormLogin);
     } else if (path.includes(PATHS.DASHBOARD)) {
       const view = new DashboardView();
-      // const controller = new DashboardController(model, view);
+      const controller = new DashboardController(model, view);
```

Comment:
DashboardController don't have any parameter. Why you put view and model into this?

Reply:
I'll update later

Reply:
Not accept this, it didn't take much time, just need move it to controller